### ISSUE TITLE
Update to 6.0.0 for Android ?

### ISSUE THREAD

**Issue by LudoFont** [viezel/NappJockey#13]
Hi,

Can you update the Android version of your (very usefull !) module to titanium SDK 6.0.0 ?
I use it in my project and want to upgrade it.

Thank's,

Ludofont

**Comment by shouse** [comment]
I will likely have to take a stab at this soon.

**Comment by caspahouzer** [comment]
Would be great, if we can use it for the actual SDK 7.1.0

**Comment by shouse** [comment]
I did indeed try to update it for 7.X and I get a null exception pointer, and couldn't trace it down further.  Instead I had to find another way around, curiously similar manner of a custom URL schema.

**Comment by JavierPascual** [comment]
Hi! @shouse, Do you know how to compile it in Android with 7.X?

**Comment by shouse** [comment]
@JavierPascual Yeah, but it didn't work... so I found a way around it and posted what I got working here https://github.com/shouse/ti.remote.webview.send-and-receive
